### DISCUSSION TITLE
从必须转发开始想 | /posts/you-must-do-it/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#627] [Announcements]
https://yuanfan.rbind.io/posts/you-must-do-it/

**Comment by botsp** [comment]
"东南西北中、党政军民学, *******" 这算是列宁式政党的特色了。
其实经济上的市场化改革，一个特点就是分散吧，分散经济权力到私营部门；
由此似乎也可以理解为，宪法议会政治是对政治权力的一种分散，票选民主来分散政治权力到不同的群体，可以理解为政治上的"市场化改革"了。

**Reply by earfanfan** [reply]
你打码的这段话，我上网搜了下，找到共产党员网的这篇：[学习问答 | 95.如何理解党政军民学，东西南北中，党是领导一切的？](https://www.12371.cn/2021/09/24/ARTI1632437750600229.shtml)。不过“党是领导一切的”这句话有撒好打码的咧，在俺们现实生活中确实就是党是领导一切的啊。

PS，想到这里也算是公众场合，最近的公众舆论给俺留下的印象是思想观念不同的人存在大范围的意识形态斗争，所以俺要多解释一下，俺说“现实生活中确实就是党是领导一切的啊”是个人眼中看到的客观事实，就是字面上的意思，没有讽刺什么，也没有赞美什么。

另外，关于上面链接的这篇文章，我发现我看得懂。现摘录几段分享与你。

>但后来计划经济时期，在“左”的错误思想影响下，加之民主与法制不健全，导致权力过分集中，政企不分、政事不分、政社不分，出现“家长制”、“一言堂”，党的一元化领导被极端化和教条化，给党和国家事业带来严重破坏。

这里描述的应该就是文化大革命时期，可以看出来这篇文章的出发点对这个特殊历史时期是有反思的，并且定的基调是带来严重破坏。只不过，这篇文章只写了“给党和国家事业带来严重破坏”，一般来说文章发出来之前都是要审稿的，也许执笔者也写过给人民生活造成严重破坏。说起来最近看新闻有不少地方大暴雨，这样的灾难在半月谈、新华社的笔法下总是只写抗灾的英雄们多么勇敢等正面信息，不怎么写灾难之下人民生活被颠覆。

>改革开放以来，我们不断深化对共产党执政规律的认识，特别是党的十八大以后，针对一段时期党的领导被忽视、淡化、弱化的状况，我们党提出坚持和加强党的全面领导，这与过去的一元化领导既有联系又有不同。

这里也提到了一段时间党的控制能力被削弱的事，若在以前看到这里我一定会忽略掉，不过现在能看得懂了。

另外，我还翻了下当前最新的[中国共产党章程](https://www.12371.cn/special/zggcdzc/zggcdzcqw/)，全文有34处提到了民主一词，看到“人民民主专政”、“民主集中制”（民主集中制是民主基础上的集中和集中指导下的民主相结合），我忽然明白我们国家的民主性质是指最高权力的合法性基础来源于人民的支持，但是民主的权力将会集中到上头，权力集中后带来专政，即便部分权力可以分配给地方，但是权力调配的方式并没有改变，因此权力并不是流通的。

你提到的民主选举算是政治上的“市场化改革”，这样理解好像也行。

**Comment by botsp** [comment]
哈哈，那俺就畅所欲言啦，
这两段摘录的特别好，算是个体感受所追溯到的官方资料了。

一段时间喊要"分开"，然后又说"被弱化"。其实"集中"可能带有天然的bug，集中到一个组织、政党，就不可避免的要集中到一群人身上，然后再集中到一个人的身上。
所以，凡是高喊着"加强党的领导"，不免令人揣测其实就是要加强个人的领导，可谓是交叉着"以人民的名义"和"以党的名义"。

**Reply by earfanfan** [reply]
其实直接把那篇文章中的“针对一段时期党的领导被忽视、淡化、弱化的状况”这句话放到网上搜搜，可以搜出来很多新的文章，比如[王岐山：弱化党的领导的状况得到根本性扭转](https://www.guancha.cn/politics/2017_11_03_433399.shtml)。我再摘录几段给你看看哈。

>在“厚植党执政的政治基础”方面，王岐山认为，改革开放以来，社会主要矛盾进化为人民日益增长的美好生活需要和不平衡不充分的发展之间的矛盾的同时，部分党员领导干部脱离群众，形式主义、官僚主义、享乐主义和奢靡之风严重，甚至违纪违法攫取利益，人民群众深恶痛绝，侵蚀了党的执政基础。

>“果断查处周永康、薄熙来、郭伯雄、徐才厚、孙政才、令计划严重违纪违法问题，铲除政治腐败和经济腐败相互交织的利益集团。”王岐山说，“5年来波澜壮阔的实践充分证明，把全面从严治党摆上战略布局英明正确，在实现伟大复兴的关键时刻，校正了党和国家事业前进的航向，使党经历了革命性锻造。”

文中提到“人民群众深恶痛绝，侵蚀了党的执政基础”，就是说腐败问题范围之广、对人民群众造成伤害之深，已经到了侵蚀党的执政基础的地步。腐败是任何国家都有的问题，伤及根本就是大问题了，但是既然问题很严重为何没能在问题变严重之前就处理呢？我猜重要原因就是这个“被忽视、弱化、淡化”，也就是以前的监察权力是在政府内部，与政府官员平级，在势力盘根错节的官僚组织里无法执行生效。

接着又有另一个问题，既然党的十八大也就是2012年就开始处理这些问题，那为撒一直等到2018年才将废除行政监察法新建立监察法，并且将监察机关从政府内部独立出来呢？就是下面那段，从2012到2017年共5年的时间里才把那一串人名代表的大势力打掉，如果你有兴趣可以查一下徐才厚案，徐是军方高层。前一篇文章里其实正好有2012-2023年发展党员与净增党员的图，从2013到2017年净增党员的数量是在减少的，尤其2017年净增党员数才11.9万，而那年发展党员数就有200万左右，可以想到那年退党的数量能有多少。

中央集权和一党专政的制度确实是有缺陷的，就连现在看起来很能起到作用的监察委与国务院平级的制度也是有缺陷的。其实我对当前的事情表象也看得不明白，不过前些时候看《明朝那些事》的时候看过一个有些微相似的故事。约是朱元璋统治明朝的时期，为了加强中央集权、更好地统御百官，设立锦衣卫监察百官，不过后来朱也意识到锦衣卫的性质容易埋下祸根于是撤销了，然而到了朱棣朝又重设了锦衣卫。但是现在和历史是不一样的，法治建设进步了许多，将来如何发展，是将来的事了。

关于各处总是提及最高领袖的名字，我觉得这也是一种制度，是我们国家政治上的威权类型接近卡理斯玛权威，即追随者忠诚追随并臣服于领袖的个人魅力。

**Comment by botsp** [comment]
刚刚注意到这个网址是12371.cn，神奇，这是除了12306以外碰到的唯一一个数字网址了。

这个网页最后一段就看了加强党的领导，然后两句话就立即转折到“…同志为核心”。

earfanfan ***@***.***>于2024年7月12日 周五10:51写道：

**Reply by earfanfan** [reply]
现在的党章第二段有这样一段话：

>中国共产党以马克思列宁主义、毛泽东思想、邓小平理论、“三个代表”重要思想、科学发展观、习近平新时代中国特色社会主义思想作为自己的行动指南。

上面每一个主义、思想、理论、观念都是每一代最高领袖在领导时期提出的，也是对本原的马克思思想的重新解读，这大概是因为党的领导的基石之一就是建立一种观念制度。不过有些思想是直接用了最高领袖的名字来命名，有些没有，其实这可以很片面地用来衡量集权的程度。从我的视角看，反复提到最高领袖的名字，并不是意在加强个人领导，而是其他。

**Comment by CyrusYip** [comment]
看你讲的动员，我就发现日常用语里面有不少军事词语。

举一下例子：

- 例句：某手机公司进军（进入）家电行业
- 例句：某手机吊打/暴打（远超）另一个手机
- 例句：某手机能打（优于其他手机）
- 老师会把考试比作战争，笔比作武器

以前我看着没什么感觉，现在从旁观者的角度看就觉得这些词语很粗暴恐怖。虽然竞争无处不在，但竞争也不是像战争那样拼到你死我活。

**Reply by earfanfan** [reply]
额，其实从党领导宣传领域开始，意识形态的东西就会持续不断地通过各类信息载体传播到每个国民的日常生活之中。而且除了军事领域的那些词语，也还有其他的，比如关键、控制、组织、纪律、强调、秩序、引起重视等等。

关于组织、纪律这两个词，我想到一个现成的例子，不知道叶寻同学你还记不记得，有一次你、我、益辉、黄国政同学在一块地方讨论秩序控相关内容，我清楚地记得益辉曾经以秩序控的视角说某些情况是文字无组织无纪律之类的。

话又说回来，由于我们国家是集权型的国家，所以日常用语中会混入这些词语，我平时不上外网，我记得你是会经常去溜达的，不知道那些倾向于分权、重视民主个人的国家的互联网络上，会不会混入的是别的特色。

**Reply by CyrusYip** [reply]
> 我清楚地记得益辉曾经以秩序控的视角说某些情况是文字无组织无纪律之类的。

这个我记得， https://github.com/earfanfan/yuanfan/pull/609#issuecomment-2001569958 。

外网给我的感觉就是政治审查比较少、有各种观点，不知道这算不算特色。

**Comment by CyrusYip** [comment]
> @所有人 福汉康参与公司所有员工必须转发上述内容到个人微信朋友圈并截图。

「福汉康参与公司」这是啥意思？这里整句话看着像病句。

**Reply by earfanfan** [reply]
福汉康只是武汉政府推出的一种惠民保性质的保险产品，实际上承保的还是商业保险公司，这里“福汉康参与公司”就是指那些参与承保的公司。
